release/ray_release/scripts/run_release_test.py

```diff
@@ -44,7 +44,7 @@
 )
 @click.option(
     "--ray-wheels",
-    default=None,
+    default="master",
```

Comment:
I think it is the reasonable default behavior. Lmk if you disagree!

Reply:
I disagree - I'd like the builds to run on the specified commit per default. This makes sense as the checks will be displayed in GitHub and e.g. trigger a red check mark on the commits (e.g. master). It makes it also easier to identify breaking commits.

It's also good for release managers working on a release branch - when they kick off the release tests for this branch they can be sure they are run on the latest commit wheels of this branch, not of some (potentially stale) wheel.

I'd be open to set "master" as the default for manually triggered runs on the master branch (e.g. pre-fill the field with "master" when kicking off).

Reply:
Isn't it possible to just always set this value if the tests are started by the build though? 

In that case, the only difference is that when you run this CLI in the local machine, you don't need to always specify `--ray-wheels master`. Or am I missing something here?

Reply:
But anyway let me merge this PR without setting the default! We can keep discussion here